src/menu.rs

```diff
@@ -155,15 +161,21 @@ pub fn run(mut window: &mut PistonWindow,
                             match menu_selection {
                                 MenuSelection::Play => {
                                     music::play(&Music::Action, music::Repeat::Forever);
-                                    game::Game::new(window_size)
-                                        .run(&mut window, &mut opengl, &mut glyph_cache);
+                                    game::Game::new(window_size).run(&mut events,
+                                                                     &mut window,
+                                                                     &mut opengl,
+                                                                     &mut glyph_cache);
                                     music::play(&Music::Menu, music::Repeat::Forever);
                                 }
                                 MenuSelection::Story => {
-                                    story::run(&mut window, &mut opengl, &mut glyph_cache);
+                                    story::run(&mut events,
```

Comment:
I just checked the docs and `GlutinWindow` has `.poll_event()` which I think should be similar to `.next()`. I think it would remove the need to have an explicit `Events` variable.